lms/static/sass/course/base/_base.scss

```diff
@@ -60,7 +60,7 @@ table {
 a {
   &:hover, &:focus {
     color: $pink;
-    text-decoration: none !important;
+    text-decoration: none;
```

Comment:
While !important is evil, I'm worried that this could impact other parts of the UI.